### PR TITLE
TRF 

### PR DESCRIPTION
**Pull request by RvDijk**
depedency for RepeatMasker

### CODE REVIEW

File: easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name = 'TRF'
+version = '4.09'
+versionsuffix = '-linux64'
+
+homepage = 'https://tandem.bu.edu/trf/trf.html'
+description = """Tandem repeats finder: a program to analyze DNA sequences. Legacy version."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+#source_urls = ['https://tandem.bu.edu/trf/trf409.linux64.download.html']
```
Comment (boegel) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: @RvDijk can this be fixed? looks like this should be `http://tandem.bu.edu/trf/downloads/`

File: easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+sources = ['trf409.legacylinux64']
+checksums = [('md5', 'f97b6d1fc06adbd1484b384d5481916a')]
+
+postinstallcmds = ["mv %(installdir)s/trf409.legacylinux64 %(installdir)s/trf"]
```
Comment (boegel) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: @RvDijk please try to avoiding hardcoding the version; replace `409` with `%(version_major)s%(version_minor)s`

Comment (RvDijk) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: @boegel %(version_minor)s seems to omit the 0 in "09", how can I include the zero?

Comment (boegel) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: hmm, that may be a bug, I would expect it to retain the `0`...

Comment (boegel) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: @RvDijk should be fixed with https://github.com/hpcugent/easybuild-framework/pull/1783, please verify?

Comment (RvDijk) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: @boegel, it took me some time to use the new framework (getting it installed etc), however it does solve this bug.

Comment (boegel) [inline easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb]: @RvDijk Thanks for the confirmation. I just released EasyBuild v2.8.1, which includes this bug fix.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/be7737cec8839c14d5c8a7bd919ce8ff for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**